src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -7,84 +8,92 @@
  * This is fired when user clicks on the fullscreen button on the menu
  */
 
-(function addFullScreenMenuToggler () {
+(function addMenuToggler () {
     jQuery.extend(BookReader.defaultOptions, {
-      enableFullScreenMenuToggle: true
+      enableMenuToggle: true
     });
   
-    function onEnterFullscreen (br) {
+    function setupNavForToggle (br) {
       var $menuTab = br.refs.$BRnav.children('.js-tooltip');
       $menuTab.css('display', 'none');
-  
-      var timeoutHandler = function onEnterTimeoutHandler () {
-        br.hideNavigation();
-        var menuManager = br.fullscreenMenu;
-  
-        registerEventHandlers(br);
-        menuManager.menuIsShowing = false;
-      }
-      setTimeout(timeoutHandler, 500);
+      registerEventHandlers(br);
     }
   
-    var onExitFullscreen = function onExitFullscreen (br) {
+    var removeToggleFromNav = function removeToggleFromNav (br) {
       var $menuTab = br.refs.$BRnav.children('.js-tooltip');
       $menuTab.css('display', 'block');
       removeEventHandlers(br);
     }
   
-    var removeEventHandlers = function removeEventHandlers(br) {
+    var removeEventHandlers = function removeEventHandlers (br) {
```

Comment:
SUGGESTION: Remove space between function name and parameters `(`?

Reply:
Run `npx eslint --fix --rule 'space-before-function-paren: [2, never]' src/js/plugins/fullscreen_menu_toggle/plugin.fullscreen_menu_toggle.js`